docs/workspaces.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Workspaces
+---
+
+## Overview
+
+Workspaces are here to allow you to collaborate with people in your team.
+Completely segreggated spaces will allow you to share kintoapps, kintoblocks and manage them in the same environment.
+
+You can invite people by email to collaborate with you on any project you may have in your workspace
```

Comment:
Note that today we do not have email confirmations nor any emails being sent for invitations, required that the user has to be registered and a validated email today in order to be successfully added.